<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Application</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkForUpdates">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="fileLocation"/>
         </item>
         <item>
          <widget class="QToolButton" name="folderSelector">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="maxRecentFiles">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Maximum Recent Files:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>File Location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>fileLocation</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Check for Updates on Startup:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Charts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Default Stitch:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>defaultStitch</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="defaultStitch"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Alternate Row Colors:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>useAltColors</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="useAltColors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="primaryColorBttn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Primary</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QPushButton" name="alternateColorBttn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Alternate</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="rowCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Rows:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>rowCount</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Default Placeholder:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>placeholder</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QComboBox" name="placeholder"/>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stitches:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>stitchCount</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QSpinBox" name="stitchCount">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Row Indicator:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>chartRowIndicator</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QPushButton" name="dotColorBttn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Indicator Color</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QComboBox" name="chartRowIndicator">
         <item>
          <property name="text">
           <string>Dots</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dots and Text</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QComboBox" name="chartStyle">
         <item>
          <property name="text">
           <string>Rows</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rounds</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Style:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>chartStyle</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="5">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Written Instructions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Generate Repeats:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>generateTextRepeats</cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QCheckBox" name="generateTextRepeats">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Color for Key words:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="keywordColorBttn">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Legends</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="6">
        <widget class="QLabel" name="label_20">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Stitch Legend&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Column Count:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>stitchLegendColumnCount</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QSpinBox" name="stitchLegendColumnCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="6">
        <widget class="QLabel" name="label_21">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Color Legend&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Color Prefix:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>colorPrefix</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="4" colspan="2">
        <widget class="QLineEdit" name="colorPrefix">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Show Border:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showColorBorder</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QCheckBox" name="showColorBorder">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Column Count:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>colorLegendColumnCount</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="4" colspan="2">
        <widget class="QSpinBox" name="colorLegendColumnCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="10" column="4" colspan="2">
        <widget class="QComboBox" name="colorLegendSortBy"/>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="colorSortByLbl">
         <property name="text">
          <string>Sort By:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>colorLegendSortBy</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="showStitchDescription">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Show Description:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showStitchDescription</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Show Border:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showStitchBorder</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="showStitchBorder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="showStitchTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Show Title:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showStitchTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Show Hex Value:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showColorHexValues</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="showColorHexValues">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="showColorTitle">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Show Title:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showColorTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="showStitchWrongSideLbl">
         <property name="text">
          <string>Show Wrong Side:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showStitchWrongSide</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="showStitchWrongSide">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="showStitchBlock">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Show Blocks:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>showStitchBlock</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkForUpdates</tabstop>
  <tabstop>fileLocation</tabstop>
  <tabstop>folderSelector</tabstop>
  <tabstop>maxRecentFiles</tabstop>
  <tabstop>defaultStitch</tabstop>
  <tabstop>rowCount</tabstop>
  <tabstop>useAltColors</tabstop>
  <tabstop>primaryColorBttn</tabstop>
  <tabstop>alternateColorBttn</tabstop>
  <tabstop>placeholder</tabstop>
  <tabstop>stitchCount</tabstop>
  <tabstop>chartStyle</tabstop>
  <tabstop>chartRowIndicator</tabstop>
  <tabstop>dotColorBttn</tabstop>
  <tabstop>showStitchTitle</tabstop>
  <tabstop>showStitchBorder</tabstop>
  <tabstop>showStitchDescription</tabstop>
  <tabstop>stitchLegendColumnCount</tabstop>
  <tabstop>showStitchBlock</tabstop>
  <tabstop>showStitchWrongSide</tabstop>
  <tabstop>showColorTitle</tabstop>
  <tabstop>showColorBorder</tabstop>
  <tabstop>showColorHexValues</tabstop>
  <tabstop>colorPrefix</tabstop>
  <tabstop>colorLegendColumnCount</tabstop>
  <tabstop>colorLegendSortBy</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAltColors</sender>
   <signal>toggled(bool)</signal>
   <receiver>primaryColorBttn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAltColors</sender>
   <signal>toggled(bool)</signal>
   <receiver>alternateColorBttn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
